Main/ClientApp/src/Components/SharedCollectionView.tsx

```diff
@@ -53,14 +58,21 @@ export default function SharedCollectionView(props: SharedCollectionViewProps) {
       {sharedCollection?.viewRights ? (
         <BaseCollectionView collection={sharedCollection?.collection}>
           {sharedCollection?.editRights && (
-            <Fab
-              color="primary"
-              aria-label="add"
-              className={classes.fab}
-              // onClick={toggleAddElementDialogOpen}
-            >
-              <AddIcon />
-            </Fab>
+            <>
+              <Fab
+                color="primary"
+                aria-label="add"
+                className={classes.fab}
+                onClick={toggleAddElementDialogOpen}
+              >
+                <AddIcon />
+              </Fab>
+              <AddElementDialog
+                toggleDialogOpen={toggleAddElementDialogOpen}
```

Comment:
Do we need the toggle method defined? Why not just put it here like in other cases?